app/controllers/api/greetings_controller.rb

```diff
@@ -0,0 +1,14 @@
+class Api::GreetingsController < ApplicationController
+  before_action :set_format
+
+  def index
+    @random_greet = Greeting.limit(1).order('RANDOM()')
+    render json: @random_greet
+  end
+
+  private
+
+  def set_format
+    request.format ||= :json
+  end
+end
```

Comment:
While you did a good job working on the requirements I noticed whenever I run your application it is not generating the random greetings . Its returning an empty array of greetings it would be nice if you fixed this issue.

This is whats being displayed when I run your application:
![Screenshot 2023-05-04 at 9 26 37 AM](https://user-images.githubusercontent.com/79506105/236138118-4bcec07a-d663-4978-b549-b9b33cc87e9c.png)